websites/V/Vizjer/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "334411435633541121",
+		"name": "meffiu"
+	},
+	"service": "Vizjer",
+	"description": {
+		"en": "Polish site with free movies and series without registration.",
+		"pl": "Polska strona z darmowymi filmami i serialami bez rejestracji."
+	},
+	"url": "vizjer.pl",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/zKFMrkt.png",
```

Comment:
Try getting a higher quality logo

Reply:
that's the highest one i could get, I even used ai to improve the quality

Reply:
gonna try again quality enhancer